### PR TITLE
Webpack 5

### PR DESCRIPTION
**Pull request by jacogr**
Closes https://github.com/polkadot-js/extension/issues/525

### CODE REVIEW

File: packages/extension/webpack.config.js [hunk 1]
```diff
@@ -18,109 +18,92 @@ const packages = [
...
-  }, {}),
-  context: __dirname
-});
+      return alias;
+    }, {}),
+    extensions: ['.js', '.jsx', '.ts', '.tsx'],
+    fallback: {
+      crypto: require.resolve('crypto-browserify'),
+      path: require.resolve('path-browserify'),
+      stream: require.resolve('stream-browserify')
```
Comment (jacogr) [inline packages/extension/webpack.config.js:105]: Adding these packages is what made the PR so big, `yarn.lock` being a bit more bloated.

File: packages/extension/webpack.config.js [hunk 2]
```diff
@@ -18,109 +18,92 @@ const packages = [
...
-    watch: !isProd
-  };
-}
-
-module.exports = createWebpack({
-  alias: packages.reduce((alias, pkg) => {
-    alias[`@polkadot/${pkg}`] = path.resolve(__dirname, `../${pkg}/src`);
+      }
+    })
+  ].filter((entry) => entry),
```
Comment (jacogr) [inline packages/extension/webpack.config.js:94]:
```suggestion
  ],
```

**Review by Tbaut — APPROVED** [review]
All good on FF, tested the following:
- import from JSON
- derive
- transfert without metadata
- upgrade metadata
- remember pw / don't
- wrong password
- forget account
- change language
- hide/show account

**Comment by jacogr** [comment]
Thank you, the second pair of eyes really help here. I just don't trust tools with new configs :)

**Pull request merged**